theme/islandora-solr-metadata-description.tpl.php

```diff
@@ -21,12 +21,12 @@
         print ($desc_array['display_label']); ?>
         <?php endif; ?></h2>
       <?php foreach($description as $value): ?>
-        <p property="description"><?php print check_markup(implode("\n", $value['value']), 'islandora_solr_metadata_filtered_html'); ?></p>
+        <p property="description"><?php print check_markup(implode(variable_get('islandora_solr_metadata_field_value_separator', "\n"), $value['value']), 'islandora_solr_metadata_filtered_html'); ?></p>
```

Comment:
Should probably set the separator as a variable in the preprocess and then use that value here as opposed to `variable_get`ing.